src/main/java/by/onliner/page/components/Menu.java

```diff
@@ -35,4 +42,38 @@ public void openShoppingCartPage() {
         shoppingCartButton.click();
         logger.info("Click 'Shopping cart' button");
     }
+
+    /***
+     * Input text in the search field
+     * @param text item what we need to find
+     */
+    @Step("Input text in the search field")
+    public void inputTextInSearchField(String text) {
+        waitForComponentOpened();
+        mainSearchField.sendKeys(text);
+        logger.info("Input [" + text + "] in the search field");
+    }
+
+    /***
+     * Get name of the first searched item
+     * @return name of searched item
+     */
+    @Step("Read name of the first searched item")
+    public String getNameOfSearchedItem() {
+        driver.switchTo().frame(driver.findElement(By.className("modal-iframe")));
```

Comment:
Move frame locator to a constant variable in this class. It might be needed for other methods

Reply:
Move not as a String but as a By

Reply:
Created WebElement of frame